### ISSUE TITLE
GameGuru Max - Enhancement - Player Character - Improved Crouching

### ISSUE THREAD

**Issue by Lemuripest93** [TheGameCreators/GameGuruRepo#3884]
I was let known by synchromesh62 via a test he did that crouching reaches 4ft 7 inches, to be able to crawl through he had to push for 5ft.

This is an enhancement request to lower the crouching height significantly enough that we could be able to crouch through openings in walls to allow for things like ventilation shafts, or other sorts of openings small enough to require crouching (such as holes in fences and other objects would also apply as a valid reasons.)

Made a tiny structure using some crappy test blocks using the default Blender cube showcasing how much of a difference in crouching height (that most likely) is required: https://www.youtube.com/watch?v=Oo5GX2oNFLk

Height reduction may not need to be reduced as significantly as seen in the video - though this could always be remedied with a slider in Player Settings called "Crouch Height" or similar, though the exact solution is up to you.

**Comment by Kitakazi** [comment]
Yes please! Cannot crouch underneath objects that appear to the player that they should easily be crouched under. 
Also RELATED: #3476
Collision capsule on the PLAYER is waaaay to big!
